Kickstarter-iOS/AppDelegate.swift

```diff
@@ -207,14 +208,14 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
 
     self.window?.tintColor = .ksr_green_700
 
-    self.viewModel.inputs.applicationDidFinishLaunching(
+    self.fbLoginViewModel.inputs.applicationDidFinishLaunching(
       application: application,
       launchOptions: launchOptions
     )
 
     UNUserNotificationCenter.current().delegate = self
 
-    return self.viewModel.outputs.applicationDidFinishLaunchingReturnValue
+    return self.fbLoginViewModel.outputs.applicationDidFinishLaunchingReturnValue
```

Comment:
I wonder if we should still return our app delegate's `applicationDidFinishLaunchingReturnValue` here? It seems that `self.fbLoginViewModel.outputs.applicationDidFinishLaunchingReturnValue` is just hardcoded to `true` 🤔

Reply:
Actually do we need this property at all? https://github.com/kickstarter/ios-oss/pull/709/files#diff-66616e30e2b9d34ca192ff1ace432180R86

It would seem that it's value should be set from the return value facebook's `didFinishLaunchingWithOptions` [here](https://github.com/kickstarter/ios-oss/pull/709/files#diff-66616e30e2b9d34ca192ff1ace432180R38), but it looks like we've always just discarded that return value.

Reply:
The only time we set this value is from the shortcut items (long-press in springboard):

```swift
self.applicationDidFinishLaunchingReturnValueProperty <~ self.applicationLaunchOptionsProperty.signal
  .skipNil()
  .map { _, options in options?[UIApplication.LaunchOptionsKey.shortcutItem] == nil }
```